Graph.py

```diff
@@ -149,6 +160,11 @@ def main():
             labels2d = Variable(labels2d)
             labels3d = Variable(labels3d)
 
+            if use_cuda and torch.cuda.is_available():
+                inputs = inputs.float().cuda(device=args.gpu_number[0])
+                labels2d = labels2d.float().cuda(device=args.gpu_number[0])
+                labels3d = labels3d.float().cuda(device=args.gpu_number[0])
+
```

Comment:
Directly below this the variables are recast as floats (but without cuda).   See my comment above for the "better" way.